ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/SubmitAndWaitRequestValidator.scala

```diff
@@ -15,13 +15,15 @@ class SubmitAndWaitRequestValidator(commandsValidator: CommandsValidator) {
 
   def validate(
       req: SubmitAndWaitRequest,
-      currentTime: Instant,
+      currentLedgerTime: Instant,
+      currentUTCTime: Instant,
```

Comment:
`currentUtcTime` -- see naming conventions for acronyms here:
https://docs.scala-lang.org/style/naming-conventions.html